pkg/manager/impl/execution_manager.go

```diff
@@ -238,7 +239,19 @@ func createTaskDefaultLimits(ctx context.Context, task *core.CompiledTask) runti
 		memoryLimit = resourceEntries[memoryIndex].Value
 	}
 
-	return runtimeInterfaces.TaskResourceSet{CPU: cpuLimit, Memory: memoryLimit}
+	taskResourceLimits := runtimeInterfaces.TaskResourceSet{CPU: cpuLimit, Memory: memoryLimit}
```

Comment:
so are we just now ignoring the request values altogether? should we remove the iteration in lines 219-240 above?

Reply:
The request values is ignored only when no limits in the config files. If no limits in the config, the behaviour is still the same as today.

Reply:
but we don't now no longer use the values of cpuLimit or memoryLimit anymore now with your change

Reply:
These values are from the computed requests values (from domain attributes/tasks/config), right? When you set the limits in the config file on purpose, what I am expecting is that these values will become the defaults limits for the task containers.

Reply:
right, that makes sense but what i'm saying is the iteration in the block above should no longer be necessary and we can delete it, right? we're not using the computed request values anymore so there's no reason to rederive them here

Reply:
So in that case, when the CPU limit is not set in the config file, the cpu default limits would be `500m`. If this is an acceptable behaviour, I am fine with deleting the block above.

Reply:
sorry if I'm not making myself clear, i mean the results of this 
```
resourceEntries := task.Template.GetContainer().Resources.Requests
	var cpuIndex, memoryIndex = -1, -1
	for idx, entry := range resourceEntries {
		switch entry.Name {
		case core.Resources_CPU:
			cpuIndex = idx

		case core.Resources_MEMORY:
			memoryIndex = idx
		}
	}

	if cpuIndex < 0 || memoryIndex < 0 {
		logger.Errorf(ctx, "Cpu request and Memory request missing for %s", task.Template.Id)
	}

	if cpuIndex >= 0 {
		cpuLimit = resourceEntries[cpuIndex].Value
	}
	if memoryIndex >= 0 {
		memoryLimit = resourceEntries[memoryIndex].Value
	}
```
were no longer being used as a result of your code changes. we shouldn't ever default to 500m but if the limits are not set for the platform i think it makes sense to use the assigned request values

Reply:
Yes. If the limits are set, this part of code should have no effect. This part was kept only to guard the situation when limits are not set. If limits are not set, the part will create default limits for us. 

If I understand correctly, when the limits are not set, `m.config.TaskResourceConfiguration().GetLimits()` will still return a dummy `limits` where `limits.CPU`== ""  and `limits.Memory`== "". So I am checking these to see if the values are set.

Reply:
nice, this looks great - thank you!